aml-development/frontend
# feat(profile): edit profile page

AMLOS-675

**Description**
Allow user to set their UI theme option from Profile page

**Acceptance Criteria**
The user will be able to set their UI theme from the Profile page.  For two options, a toggle switch would work, but for three or more, maybe a dropdown.

**How To Test**
Pull `edit_profile_page` from `frontend`
1.  Login and go to the profile page from the navigation bar
2.  Edit your profile from the profile header (pencil icon on the right)
3.  In the _Preferences_ section, select a theme from the drop-down.  Right now, there are only two options: Light Theme and Dark Theme
4.  Save changes by clicking the save icon in the profile header
5.  Ensure that the site's theme updates and you are redirected to your profile page.
6.  Ensure that refreshing the page preserves your selected theme.
7.  Ensure that editing your profile has the updated theme as the selected item in the drop-down.

NOTE: for now, only the site theme can be updated when editing a profile.  The remaining options are there, but not all of them are filled from the profile or update.
